lib/moon_web/components/header.ex

```diff
@@ -0,0 +1,51 @@
+defmodule MoonWeb.Components.Header do
+  @moduledoc false
+
+  use MoonWeb, :stateless_component
+
+  prop click, :event
+  slot default
+
+  def render(assigns) do
+    ~F"""
+    <header>
+      <div class="relative z-10 py-4 lg:hidden flex flex-row gap-2 items-center justify-between text-bulma">
+        <button type="button" class="focus:outline-none" phx-click={@click}>
+          <span class="sr-only">Open sidebar</span>
+          <svg
+            width="1em"
+            height="1em"
+            viewBox="0 0 32 32"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            class="text-moon-32"
+          >
+            <path d="M7.5 11.5h17m-17 8h17" stroke="currentColor" stroke-linecap="round" />
+          </svg>
+        </button>
+        <a href="/">
+          <svg
+            width="1em"
+            height="1em"
+            viewBox="0 0 24 24"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            font-size="2rem"
+            class="sc-eqhNQH fcnWEQ"
```

Comment:
Why SVG is inline not in separate file?

Reply:
i need to pass a connection/socket property  through 3 levels to get static path for theese svgs. looks like it's more complicated. what do you think?

Reply:
is there a simlier way to get static path than `{static_path(@socket, "/svgs/moon_web/moon_logo.svg")}` ?